routes/onGuess.go

```diff
@@ -0,0 +1,60 @@
+package routes
+
+import (
+	i "johnSamilin/golang-hangman/interfaces"
```

Comment:
it is usually uncommon to create an import alias just because of a long name. Save this feature for complex package names or duplicates. `interfaces` is a fine package name to have.